macroid-core/src/main/scala/macroid/Contexts.scala

```diff
@@ -45,12 +46,19 @@ object ContextWrapper {
 
   def apply[F](fragment: F)(implicit fragmentImpl: Fragment[F]): ActivityContextWrapper =
     ContextWrapper(fragmentImpl.activity(fragment))
+
+  def apply(view: View): ContextWrapper = view.getContext match {
+    case activity: Activity => ContextWrapper(activity)
+    case service: Service => ContextWrapper(service)
+    case app: Application => ContextWrapper(app)

```

Comment:
Good catch @pfn !

I didn't remember that. Thank you

I think that we should use DefaultContextWrapper approach. I'm going to wait  to the opinion of @stanch about that